app/views/wiki/show.html.erb

```diff
@@ -22,6 +22,10 @@
 
 <div class="row">
 
+<% if @node.latest.created_at < Time.now - 6.months %>
+  <div class="alert alert-info" role="alert">This wiki page hasn't been edited for quite long. Consider updating it if you are acquainted with it</div>
```

Comment:
Could we have this text say:

`This wiki page hasn't been edited for a while. If you see anything that needs updating, please help out!`

Thank you, and apologies for changing the requested text.